pkg/data/secret.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+import (
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/harvester/harvester/pkg/config"
+	"github.com/harvester/harvester/pkg/util"
+)
+
+func createSecrets(mgmt *config.Management) error {
+	secrets := mgmt.CoreFactory.Core().V1().Secret()
+
+	// Initializing the secret for Plan cattle-system/sync-additional-ca,
+	// so sync-additional-ca doesn't fail to mount harvester-additional-ca secret to jobs.
+	additionalCA := corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: cattleNamespace,
```

Comment:
Could we reuse this?

```suggestion
			Namespace: util.CattleSystemNamespaceName,,
```

Reply:
Updated it. Thank you.